applications/RANSApplication/python_scripts/formulations/fractional_step/fractional_step_velocity_pressure_rans_formulation.py

```diff
@@ -0,0 +1,344 @@
+
+        self.solver_settings.SetStrategy(
+            strategy_label_type.Velocity,
+            velocity_linear_solver,
+            settings["velocity_tolerance"].GetDouble(),
+            settings["maximum_velocity_iterations"].GetInt())
+        self.solver_settings.SetStrategy(
+            strategy_label_type.Pressure,
+            pressure_linear_solver,
+            settings["pressure_tolerance"].GetDouble(),
+            settings["maximum_pressure_iterations"].GetInt())
+
+        if self.IsPeriodic():
+            self.solver = GetKratosObjectType("FractionalStepStrategy")(
+                self.fractional_step_model_part,
+                self.solver_settings,
+                settings["predictor_corrector"].GetBool(),
+                False,
+                KratosCFD.PATCH_INDEX)
+        else:
+            self.solver = GetKratosObjectType("FractionalStepStrategy")(
+                self.fractional_step_model_part,
+                self.solver_settings,
+                settings["predictor_corrector"].GetBool(),
+                False)
+
+        process_info.SetValue(Kratos.DYNAMIC_TAU, settings["dynamic_tau"].GetDouble())
+        process_info.SetValue(Kratos.OSS_SWITCH, settings["oss_switch"].GetInt())
+
+        super().Initialize()
+
+        Kratos.Logger.PrintInfo(self.GetName(), "Solver initialization finished.")
+
+    def GetMinimumBufferSize(self):
+        return self.min_buffer_size
+
+    def IsConverged(self):
+        if (hasattr(self, "is_converged")):
+            if (self.is_steady_simulation):
+                settings = self.GetParameters()["steady_convergence_settings"]
+                formulation_converged = self.__CheckTransientConvergence(
+                    Kratos.VELOCITY,
+                    settings["velocity_tolerances"])
+                self.is_converged = self.is_converged and formulation_converged
+
+                formulation_converged = self.__CheckTransientConvergence(
+                    Kratos.PRESSURE,
+                    settings["pressure_tolerances"])
+                self.is_converged = self.is_converged and formulation_converged
+
+            return self.is_converged
+        return False
+
+    def SolveCouplingStep(self):
+        if (self.IsBufferInitialized()):
+            max_iterations = self.GetMaxCouplingIterations()
+            for iteration in range(max_iterations):
+                self.solver.Predict()
+                self.is_converged = self.solver.SolveSolutionStep()
+                self.ExecuteAfterCouplingSolveStep()
+                Kratos.Logger.PrintInfo(self.GetName(), "Solved coupling iteration " + str(iteration + 1) + "/" + str(max_iterations) + ".")
+                return True
+
+        return False
+
+    def InitializeSolutionStep(self):
+        if (self.IsBufferInitialized()):
+            super().InitializeSolutionStep()
+
+    def FinalizeSolutionStep(self):
+        if (self.IsBufferInitialized()):
+            super().FinalizeSolutionStep()
```

Comment:
Why these are required? I think they can be removed to directly call the base ones.

Reply:
done :)